dired-subtree.el

```diff
@@ -276,18 +276,21 @@ If no SUBTREES are specified, use `dired-subtree-overlays'."
   "Unmark a file without moving point."
   (save-excursion (dired-unmark 1)))
 
+(defun dired-line-is-directory-or-link-p ()
```

Comment:
This needs to have a `dired-subtree--` prefix for the name.  That's the emacs convention to simulate modules.  The two dashes signify a private function.

Reply:
Will do